document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -275,6 +275,11 @@ public BulkUpdateResult bulkOperationOnArrayValue(BulkArrayValueUpdateRequest re
     return Collections.emptyIterator();
   }
 
+  @Override
+  public Iterator<Document> aggregate(Query query) {
```

Comment:
will this break postgres HT setup if this support is not implanted on Postgres side?

Reply:
It would not as we are introducing a new API, which we would be consuming later.